doc/source/start.rst

```diff
@@ -31,6 +31,13 @@ If you want to include TOML support, install it with the `toml` extras:
 
     pip install bandit[toml]
 
+If you want to use the bandit-baseline CLI, install it with the `GitPython`
+extras:
+
+.. code-block:: console
+
+    pip install bandit[GitPython]
```

Comment:
I didn't properly object before to the usage of `toml` as the extra name, that should have been `pyproject`. But I don't think bleeding implementation details of functionality behind extras is the right decision either. This should be `bandit[baseline]` or `bandit[baseline-deps]`